modules/libcom/src/misc/epicsString.h

```diff
@@ -24,20 +30,107 @@
 extern "C" {
 #endif
 
+/** \brief Converts C-style escape sequences to their binary form
+ * 
+ * Copies characters from string to an output buffer and converts C-style escape sequences to 
+ * their binary form.  Since the output string can never be longer than the source, it is legal for 
+ * @p inbuf and @p outbuf to point to the same buffer and @p outsize and @p inlen
+ * be equal, thus performing the character translation in-place.
+ * 
+ * \param outbuf  buffer to copy string to. The resulting string will be zero-terminated as long as 
+ *                @p outsize is non-zero. 
+ * \param outsize length of output buffer not including the null-terminator. 
+ * \param inbuf   buffer to copy from. Null byte terminates the string.
+ * \param inlen   maximum number of characters to copy from input buffer.
+ *
+ * \return  number of characters that were written into output buffer, not counting the null terminator. 
+**/
 LIBCOM_API int epicsStrnRawFromEscaped(char *outbuf,      size_t outsize,
                                            const char *inbuf, size_t inlen);
+
+/** \brief Converts non-printable characters into C-style escape sequences
+ *
+ * Copies characters from the string into a output buffer converting non-printable characters into C-style 
+ * escape sequences.   In-place translations are not allowed since the escaped results will usually be larger
+ * than the input string.
+ *
+ * The following escaped character constants will be used in the output:
+ * \a \b \f \n \r \t \v \\ \’ \" \0
```

Comment:
Some characters will need to be escaped as they will be interpreted as a Doxygen command (for example `\a` is the Doxygen command for [italics](https://www.doxygen.nl/manual/commands.html#cmda)). It should be possible to escape via the backslash via `\\`, or by surrounding the section with `\verbatim`, `\endverbatim` ([doc](https://www.doxygen.nl/manual/commands.html#cmdverbatim)). The last option is better in my opinion since it allows reading the source code as is.

Reply:
Doxygen comment has been fixed.